src/ScriptBuilder/Outbox/Create_MsSqlServer.sql

```diff
@@ -32,23 +32,6 @@ begin
   declare @createDispatchedAtIndex nvarchar(max);
   set @createDispatchedAtIndex = '
   create index Index_DispatchedAt
-  on ' + @tableName + '(DispatchedAt);';
+  on ' + @tableName + '(DispatchedAt) where Dispatched = 1;';
```

Comment:
So, this is a [filtered index](https://docs.microsoft.com/en-us/sql/relational-databases/indexes/create-filtered-indexes), which was new in SQL 2016, and we [currently support down to SQL 2012](https://docs.particular.net/nservicebus/sql-persistence/?version=sqlpersistence_2#supported-sql-implementations). :frown: